ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -70,9 +73,15 @@ spec:
       description: >-
         The key within the Kubernetes Secret that contains the kerberos keytab for submitting IIB builds.
       default: krb5.keytab
-    - name: metrics_endpoint
-      description: Prometheus metrics endpoint
-      default: http://pipeline-metrics.pipeline-metrics-prod
+    - name: umb_ssl_secret_name
```

Comment:
Is this required? UMB is using the same cert as Puxis. Can we just reuse it?

Reply:
We can reuse it sure. I was thinking it'll give some flexibility if in the future a different cert is used, but that's unlikely and we can always add the params in the future as needed. Updated

Reply:
Updated for real, please check again.